config/src/config/storage_config.rs

```diff
@@ -18,6 +18,7 @@ pub struct RocksdbConfig {
     pub max_total_wal_size: u64,
 }
 
+#[allow(clippy::integer_arithmetic)]
```

Comment:
you need this for `1u64 << 30`? Clippy shouldn't warn on this : o we probably should report that to clippy and add a TODO to remove this once this is fixed upstream

Reply:
sorry to restart this ^^ but can you add this macro to the actual operation itself? And can you add a TODO that we should remove this once fixed by clippy?

Reply:
sorry missed this, yes, solved!